src/dialects/abstract/query-interface.d.ts

```diff
@@ -319,7 +320,7 @@ export class QueryInterface {
    *
    * @param options
    */
-  showAllSchemas(options?: QueryRawOptions): Promise<object>;
+  showAllSchemas(options?: ListSchemasQueryOptions): Promise<string[]>;
```

Comment:
`ListSchemasQueryOptions` is only the options accepted by queryGenerator, but the options are also passed to queryRaw. This interface should accept both